src/executor_factory.py

```diff
@@ -76,15 +82,19 @@ def create_executor(
         user_query = (
                 general_guidelines()
                 + executor_file_task(executor_name, description, test, package)
-                + chain_of_thought_creation()
+                + '\n\n' + chain_of_thought_creation()
```

Comment:
see if we can skip `if executor_content == '':` by improving `chain_of_thought_creation` and `executor_file_task`

Reply:
chain_of_thought_creation improved...
I leave the if condition. it does not create too much pollution that it is there. So the potential benefit is larger from my perspective